inc/Messages/MessageId.h

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2018 United States Government as represented by the
+// Administrator of the National Aeronautics and Space Administration.
+// All Rights Reserved.
+#ifndef PCOE_MESSAGEID_H
+#define PCOE_MESSAGEID_H
+#include <cstdint>
+
+namespace PCOE {
+    /**
+     * The set of pre-defined message identifiers. Any integer in the range of
+     * uint64_t may be a valid identifier, any entity dealing with messages should
+     * not treat this list as exhaustive.
+     *
+     * @author Jason Watkins
+     * @since 1.2
+     **/
+    enum class MessageId : std::uint64_t {
+        // Note (JW): If you add a value to this enum, add it to the map in
+        //            MessageId.cpp as well.
+        All = 0x0000000000000000L,
+        Voltage = 0x6272000000000000L,
+        Current = 0x6272000000000100L,
+        Power = 0x6272000000000200L,
```

Comment:
Can we add units to the enum for all values that have units like you do for temperature (e.g., Power_W to differentiate from a future Power_Hp)?

Reply:
Sadly, I think you're right. I would propose that we treat SI units as special, so "Voltage" means the SI unit of electrical potential as expected, "Power" means "power in watts", and rename "Temperature_K" to just "Temperature" while leaving "Temperature_C" and "Temperature_F" as they are. Thoughts?

Reply:
Also, it occurs to me that I'm mixing terms in this enum. I should either rename "Voltage" to "ElectricalPotential" and "Current" to "Amperes" or rename "Power" to "Wattage". Not sure which way to go on that.